code/shared/os/Firewall.cpp

```diff
@@ -0,0 +1,50 @@
+#include "Firewall.h"
+
+namespace fx::os
+{
+std::wstring Firewall::GetActiveName(std::wstring_view defaultName)
+{
+#ifdef _WIN32_WINNT >= 0x0602 // Need at least windows 8 to compile this
+
+	using namespace Microsoft;
+
+	// IWSCProductList and IWscProduct are Windows 8+ Desktop
+	if (IsWindows8OrGreater())
```

Comment:
We only support windows 10+ afaik.

Reply:
Correct, I left it in as it was from the old code